client/my-sites/customer-home/cards/features/quick-start/index.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { Button, Card } from '@automattic/components';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import HappinessEngineersTray from 'components/happiness-engineers-tray';
+import CardHeading from 'components/card-heading';
+import {
+	withAnalytics,
+	composeAnalytics,
+	recordTracksEvent,
+	bumpStat,
+} from 'state/analytics/actions';
+import { navigate } from 'state/ui/actions';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import QueryConciergeInitial from 'components/data/query-concierge-initial';
+import getConciergeNextAppointment from 'state/selectors/get-concierge-next-appointment';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const QuickStartCard = ( {
+	translate,
+	siteId,
+	siteSlug,
+	nextSession,
+	bookASession,
+	viewDetails,
+	reschedule,
+} ) => {
+	return nextSession ? (
+		<Card className="quick-start next-session">
+			<HappinessEngineersTray />
+			<CardHeading>{ translate( 'Your scheduled Quick Start support session' ) }</CardHeading>
+			<div className="quick-start__date">
+				{ translate( 'Date' ) }
+				{ nextSession.beginTimestamp }
+			</div>
+			<div className="quick-start__time">
+				{ translate( 'Time' ) }
+				{ nextSession.beginTimestamp }
+			</div>
+			<Button onClick={ () => viewDetails( siteSlug ) }>{ translate( 'View details' ) }</Button>
+			<Button onClick={ () => reschedule( siteSlug, nextSession.id ) }>
+				{ translate( 'Reschedule' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Reschedule session' )` *ES Score: 8*